che-tomcat8-slf4j-logback/src/assembly/conf/logback.xml

```diff
@@ -21,35 +21,12 @@
 
     <jmxConfigurator/>
 
-    <appender name="stdout" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>
-            <pattern>%-41(%date[%.15thread]) %-45([%-5level] [%.30logger{30} %L]) - %msg%n</pattern>
-        </encoder>
-    </appender>
-
-    <appender name="file" class="ch.qos.logback.core.rolling.RollingFileAppender">
-        <append>true</append>
-        <prudent>true</prudent>
-        <encoder>
-            <charset>utf-8</charset>
-            <pattern>%-41(%date[%.15thread]) %-45([%-5level] [%.30logger{30} %L]) - %msg%n</pattern>
-        </encoder>
-        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-            <fileNamePattern>${che.logs.dir}/archive/%d{yyyy/MM/dd}/catalina.log</fileNamePattern>
-            <maxHistory>${max.retention.days}</maxHistory>
-        </rollingPolicy>
-    </appender>
-
-
     <include optional="true" file="${che.local.conf.dir}/logback/logback-additional-appenders.xml"/>
     <include optional="true" file="${catalina.home}/conf/logback-additional-appenders.xml"/>
+    <!--Include appenders file depending on the variable `che.logs.appenders_impl` -->
```

Comment:
s/che.logs.appenders_impl/CHE_LOGS_APPENDERS_IMPL/ in the line above ?

Reply:
thanks